02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,60 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Steven's Webpage</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
-<body>
+
+<body background="https://images.unsplash.com/photo-1422228527100-2ecacc4c92a3?dpr=1&auto=format&crop=entropy&fit=crop&w=1500&h=1120&q=80"/>
+  <header>
+    <img src="https://pbs.twimg.com/profile_images/425948730/DF-Star-Logo.png" align="left"/>
+    <img src="https://pbs.twimg.com/profile_images/425948730/DF-Star-Logo.png" align="right"/>
+    <h1>Steven Fiero</h1>
+    <h4 style="color:white">Austin, TX 78717</h4>
+    <h4><a href="cell:512-619-1209">Contacts: Cell - </a><a href="email:ssfiero@yahoo.com">Email</a></h4>
+  </header>
+
+  <h1 style="color:#056F87">Web Wizard in-training</h1>
+
+  <h3>This is my Bio</h3>
+    <p>I was born and raised in Austin, TX - and I am loving coding.  Coding allows me to be<br>a builder and use my creative chops.  I am currently working on developing my HTML and<br>CSS skills.  I have a long and stable work history consulting in the telecom industry,<br>where I developed my hardware knowledge.  I also have a background in business finance.<br>Check out more below!</p>
+
+  <h3>Interests</h3>
+    <ul>
+      <li>Coding - Designing awesome webpages</li>
+      <li>Sports - Football, Golf, Baseball, Fantasy sports</li>
+    </ul>
+
+  <h3>A little more about myself</h3>
+    <ul>
+      <li>Work full-time as Telecom Consultant</li>
+      <li>Texas State Grad - Business Finance degree</li>
+      <li>Enjoy spending time with family - wife, daughter, dogs</li>
+    </ul>
+
+    <table align="right">
+      <thead>
+        <tr>
+          <th col colspan="5">Cool Websites</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><a href="http://www.aclfestival.com/"/><img src="http://aclp3.c3cdn.com/wp/wp-content/uploads/2016/02/ACL16-Facebook-Share-Image-1200x629v2.jpg"/></td>
+          <td><a href="http://www.wired.com"/><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/95/Wired_logo.svg/2000px-Wired_logo.svg.png"/></td>
+          <td><a href="http://www.austincodingacademy.com/"/><img src="https://i.ytimg.com/vi/29wiPl5vKos/maxresdefault.jpg"/></td>
+          <td><a href="http://austinartgarage.com/"/><img src="https://pbs.twimg.com/profile_images/3603562132/a370e03a7f8b9003deffbc176fd8da81_400x400.jpeg"/></td>
+          <td><a href="http://www.txstate.edu"/><img src="http://d54ha93qrghds.cloudfront.net/variant/large/otter-samsung-galaxy-s6-com/black/texas-state-design1_color_3.jpg"/></td>
+        </tr>
```

Comment:
fantastic use of table and background image. everywhere i looked there was something to see.